static/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -387,7 +387,7 @@ class SpanDetail extends Component<Props, State> {
                   fixed: 'Mar 16, 2020 9:10:12 AM UTC',
                   value: (
                     <Fragment>
-                      <DateTime date={startTimestamp * 1000} />
+                      <DateTime date={startTimestamp * 1000} year seconds timeZone />
```

Comment:
We still want to show the full time string here, since it's a data field.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/44172267/173710664-685b2e1b-09f6-46ff-8eaf-02c1e4bd77da.png">